jsep.go

```diff
@@ -28,6 +29,16 @@ const (
 	SemanticTokenWebRTCMediaStreams     = "WMS"
 )
 
+// Constants for extmap key
+const (
+	ExtMapValueTransportCC = 3
+)
+
+// ExtMapURI is a map
+var ExtMapURI = map[int]string{
```

Comment:
Does this need to be public? I would maybe make a function that returns the map, right now it is possible for users to modify the `ExtMapURI` which could break things

Reply:
will fix, like: src/net/http/status.go